site/content/docs/commands/task-delete.md

```diff
@@ -0,0 +1,43 @@
+# task delete
+```
+$ copilot task delete
+```
+
+## What does it do?
+`copilot task delete` removes the resources created by a one-off task.
+
+Generally, the steps involved in `task delete` are:
+
+1. Stop all running instances of the task.
+2. Empty the ECR repository for the task.
+3. Delete the CloudFormation stack, including the ECR repo, log group, and ECS Task Definition.
+
+!!!info
+    1. Tasks created with versions of Copilot earlier than v1.2.0 cannot be stopped by `copilot task delete`. Customers using tasks launched with earlier versions should manually stop any running tasks via the ECS console after running the command. 
```

Comment:
What was the reason for this?

Reply:
Previously, the call in `task run` to `ecs.RunTask` did not have the `PropagateTags: aws.String(ecs.PropagateTagsTaskDefinition)` property set so the tasks cannot be reliably identified as Copilot-initiated.

Reply:
The side effect would be the ability to stop tasks from a one-off task definition not started by copilot.

Reply:
ooh yeaa gotcha, thanks for the clarification